README.md

```diff
@@ -158,6 +163,24 @@ https://stackoverflow.com/questions/49197916/how-to-profile-cpu-usage-of-a-pytho
 Done. All URLS passed.
 ```
 
+If we want to filter down file types (for example, to only check different file
+types) we might do any of the following:
+
+```bash
+# Check only html files
+urlchecker check --file-types *.html .
+
+# Check hidden flies
+urlchecker check --file-types ".*" .
+
+# Check hidden files and html files
+urlchecker check --file-types ".*,*.html" .
+```
+
+Note that while some patterns will work without quotes, it's recommended for most
```

Comment:
suggestion: can we make this important note bold or italic? 
what do you think?

Reply:
Sure thing, done.